source/api/index.md

```diff
@@ -42,6 +42,17 @@ __Translation Note__<br/>
 Please note that the IIIF community does not guarantee the accuracy of any translation. They are linked to for information purposes only, and any discrepancies with the specifications are unintentional. The English versions of the specifications linked above are the definitive versions.
 {: .alert}
 
+## Community Cookbook
+
+The [cookbook][cookbook] gathers together examples of how to create IIIF Presentation API assets, in order to:
+
+ * Provide many more examples than the specification alone can do, for reference and learning;
+ * Encourage publishers to adopt common patterns in modeling classes of complex objects;
+ * Enable client software developers to support these patterns, for consistency of user experience (when desirable);
+ * Demonstrate the applicability of IIIF to a broad range of use cases.
+
+The community welcomes additions to the [cookbook][cookbook] and to get started have a look at the cookbook process and say hello on the cookbook [slack][slack] channel.
```

Comment:
I would edit this slightly to "The community welcomes additions to the [cookbook][cookbook]. To get started, review the [cookbook process] [link to process?] and say hello on the cookbook [slack][slack] channel."

Reply:
Thanks, made that change.